src/main/webapp/resources/js/components/samples/SampleDetailViewer.tsx

```diff
@@ -54,7 +54,9 @@ function DisplaySampleDetails({
       skip: !visible,
     }
   );
-  const { sampleIds } = useSelector((state) => state.cartSamplesReducer);
+  const { sampleIds } = useSelector(
+    (state: RootStateOrAny) => state.cartSamplesReducer
```

Comment:
Update this

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f